doc/source/credits.rst

```diff
@@ -96,10 +99,14 @@ Bradley Dice - **Lead developer**
 * Added some freud tests.
 * Added ReadTheDocs support.
 * Rewrote interface module into pure Cython.
-* Proper box duck-typing.
+* Added box duck-typing.
 * Removed nose from unit testing.
 * Use lambda function for parallelizing CorrelationFunction with TBB.
 * Finalized boost removal.
+* Wrote AABBQuery class.
+* Consolidated cluster module functionality.
+* Rewrote SolidLiquid order parameter class.
+* Updated AngularSeparation class.
```

Comment:
Either you or Yezhi should be added for the new Voronoi, unless you're waiting on the even newer implementation using Voro++. Also things for Minkowski structure metrics.